pkg/addon/helper.go

```diff
@@ -91,8 +132,61 @@ func GetAddonStatus(ctx context.Context, cli client.Client, name string) (Status
 	}
 }
 
+// GetObservabilityAccessibilityInfo will get the accessibility info of addon in local cluster and multiple clusters
```

Comment:
```suggestion
// GetObservabilityAccessibilityInfo will get the accessibility info of an addon from local and children clusters
```

Reply:
@hongchaodeng Thanks for this change. I have addressed it in https://github.com/oam-dev/kubevela/pull/2974.